old/.gitignore

```diff
@@ -0,0 +1,10 @@
+### Sass ###
```

Comment:
You can remove this file since we are not working with these files anymore :ok_hand: